oshmem/mca/sshmem/ucx/sshmem_ucx_module.c

```diff
@@ -126,6 +126,10 @@ segment_create_internal(map_segment_t *ds_buf, void *address, size_t size,
     mem_map_params.length     = size;
     mem_map_params.flags      = flags;
 
+    if (spml->symmetric_rkey) {
+        mem_map_params.flags |= UCP_MEM_MAP_SYMMETRIC_RKEY;
```

Comment:
pls add configure check - it's a new flag

Reply:
added configure and wrapper function to have flag in one place.